lms/djangoapps/teams/static/teams/js/views/instructor_tools.js

```diff
@@ -25,14 +29,46 @@
 
                 deleteTeam: function (event) {
                     event.preventDefault();
-                    alert("You clicked the button!");
-                    //placeholder; will route to delete team page
+                    var self = this;
+                    var msg = new PromptView.Warning({
+                        title: gettext('Delete this team?'),
+                        message: gettext('Deleting a team removes it from the team listing view, and removes members from the team as well.'),
+                        actions: {
+                            primary: {
+                                text: gettext('Delete'),
+                                class: 'action-delete',
+                                click: function () {
+                                    msg.hide();
+                                    self.handleDelete();
+                                }
+                            },
+                            secondary: {
+                                text: gettext('Close'),
+                                class: 'action-cancel',
+                                click: function () {
+                                    msg.hide();
+                                }
+                            }
+                        }
+                    });
+                    msg.show()
                 },
 
                 editMembership: function (event) {
                     event.preventDefault();
                     alert("You clicked the button!");
                     //placeholder; will route to remove team member page
+                },
+
+                handleDelete: function () {
+                    var self = this;
+                    this.team.destroy().then(function () {
```

Comment:
I don't think we have a wireframe that conveys a deletion error. I'm fine with exploring the use of the teams error message on the topic page to convey deletion error (or maybe even deletion confirmation.) That being said, if possible this might be better handled as a follow-on story to make sure we're doing these messages as we'd like to.  I'll defer to @explorerleslie on what she'd like to do here.